scripts/homebrew/homebrew_formula.yml

```diff
@@ -24,7 +24,7 @@
 #@ def download_info(os, arch, product, version, github_url):
 #@ if has(os, arch):
   #@ sha256 = get_shasum(os, arch)
-  #@ return 'url "https://github.com/vmware-tanzu/carvel-{product}/releases/download/{version}/{product}-{os}-{arch}"\n      sha256 "{sha256}"'.format(product=product, version=version, sha256=sha256, os=os, arch=arch)
+  #@ return 'url "{github_url}/releases/download/{version}/{product}-{os}-{arch}"\n      sha256 "{sha256}"'.format(product=product, version=version, sha256=sha256, os=os, arch=arch)
```

Comment:
@joaopapereira I am hoping that this would work, but is there any way to test it?

Reply:
We do not have tests for this, but the only way to do it is to mimic this call https://github.com/vmware-tanzu/carvel-release-scripts/blob/main/.github/workflows/published_release.yml#L126
You can create a simple releases.yml file that is static and run the script and check if there is any diff in the files. Since it uses ytt under the hood it should give you always the same  result

Reply:
I will try to test it using this method. Thanks 😁

Reply:
Tested it made changes accordingly. But still need to update the homepage url.

Reply:
Done.